src/screens/Heading.js

```diff
@@ -156,6 +156,19 @@ const HeadingPage = () => (
           </Description>
           <GenericBoolFalse />
         </Property>
+
+        <Property name="weight">
+          <Description>Font weight</Description>
+          <PropertyValue type="string">
+            <Example>"normal"</Example>
```

Comment:
```suggestion
            <Example defaultValue>"normal"</Example>
```
I assume normal weight is the default value here